apps/platform-app/src/app/[locale]/(profile)/(components)/VerificationModal/VerificationModal.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+
+import { getProfileByUserId } from '@/backend/profile/profile.service'
+import { Button } from '@gdh/ui-system'
+import { PublishingState } from '@prisma/client'
+import styles from './VerificationModal.module.scss'
+import { GithubIcon } from '@gdh/ui-system/icons'
+
+interface VerificationModalProps {
+  profileStatus: PublishingState;
+  onClose: () => {}
```

Comment:
**nitpick**: The `onClose` function type should be `() => void` instead of `() => {}` for better type clarity.